tests/callbacks/test_early_stopping.py

```diff
@@ -56,6 +56,7 @@ def test_resume_early_stopping_from_checkpoint(tmpdir):
     https://github.com/PyTorchLightning/pytorch-lightning/issues/1464
     https://github.com/PyTorchLightning/pytorch-lightning/issues/1463
     """
+    tmpdir = str(tmpdir)
```

Comment:
Can it be done in conftest.py directly ?

Reply:
Tangentially related, what if we converted all `tmpdir` inputs to `pathlib.Path` in `conftest`?

Reply:
it is a `py.path.LocalPath` not a `pathlib.Path`, there are small differences but the latter is much more common

Reply:
https://stackoverflow.com/a/67027860